Classes/Views/HangingChadItemWidth.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  HangingChadItemWidth.swift
+//  Freetime
+//
+//  Created by Ryan Nystrom on 1/1/18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+
+func HangingChadItemWidth(
```

Comment:
What is / was this for? Seems to be unused.

Reply:
It is now, used to be for spacing the action buttons evenly. We can kill it.